lib/plugins/spawn/compileServiceCode/java/index.js

```diff
@@ -0,0 +1,30 @@
+var tmp = require('tmp');
+var path = require('path');
+var fs = require("fs");
+var spawnSync = require('child_process').spawnSync;
+
+function getClassNameFromSource(code) {
```

Comment:
@peterwilli - Do you think this is the best idea? I'm considering enforcing a fixed exported class name for all Java services. Maybe `hook` or `service` or `microcule`